frameworks/g-cloud-9/questions/services/serviceCategoriesHosting.yml

```diff
@@ -2,15 +2,39 @@ id: serviceCategories
 name: Service categories
 question: Which categories does your service fit under?
 hint: You can choose multiple categories.
-
 optional: true
 depends:
   - "on": lot
     being:
       - cloud-hosting
-
-type: checkboxes
+type: checkbox_tree
 options:
-  - label: Hosting type 1
-  - label: Hosting type 2
-  - label: Or maybe hosting doesn’t have categories at all?
+- {label: 'Archiving, backup and disaster recovery'}
```

Comment:
These were generated automatically by PyYaml (presumably it has some algorithm about keeping to one line if possible). I'm a bit reluctant to add code to tweak the formatting, but no, the formatting doesn't matter to us in any way.